app/src/main/java/com/gophillygo/app/activities/MapsActivity.java

```diff
@@ -56,13 +56,15 @@
     private static final int DEFAULT_ZOOM = 12;
     private static final int ATTRACTION_ZOOM = 14;
     private static final float DEFAULT_OPACITY = 1f, FILTERED_OPACITY = 0.5f;
+    private static final String LOCATION_SET_KEY = "locationSet";
     private static final String LOG_LABEL = "MapsActivity";
+
     public static final String X = "x";
     public static final String Y = "y";
     public static final String ATTRACTION_ID = "id";
 
     private Map<Integer, Marker> markers;
-    private Marker selectedMarker;
+    private Integer selectedAttractionId = null;
```

Comment:
How about we make this a primitive and default for -1 as a flag value for being unset; that's what we're doing elsewhere.